docs/source/getting-started.mdx

```diff
@@ -102,19 +106,36 @@ Finally we'll add a `type` and `script` to the `package.json` file you previousl
 }
 ```
 
-Setting your project's type to `module` will enable you to use a top-level [`await`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function), which can make your code easier to read.
+The above `start` script tells TypeScript to compile your code into JavaScript before using `node` to run that compiled code. Setting your project's [`type` to `module`](https://nodejs.org/api/packages.html#approach-1-use-an-es-module-wrapper) loads your JavaScript files as ES modules, enabling you to use top-level [`await`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) calls.
+
+</ExpansionPanel>
+
+If you are using JavaScript (i.e., not TypeScript), add the following script to your `package.json`:
+
+```json title="package.json"
+{
+  "scripts": {
+    "start": "node index.js"
```

Comment:
Note, up above we told JS users to create a `src/index.js`, so this command won't work. (we should probably tell JS users not to have a `src` folder at all, that's a TS convention)